packages/runtime/lib/schema.js

```diff
@@ -27,6 +27,7 @@ const services = {
       },
       path: {
         type: 'string',
+        allowEmptyPaths: true,
```

Comment:
Why is this? I don't think this is documented

Reply:
document it

Reply:
Will do